src/elements/select/select.svelte

```diff
@@ -241,6 +251,10 @@ const handleClearAll = () => {
   }
 };
 
+const handleButtonClick = () => {
+  dispatch('buttonclick');
```

Comment:
nit: typically we do multi-word events with a dash like 'button-click'

Reply:
okie!